app/helpers/patient_details_helper.rb

```diff
@@ -160,4 +160,8 @@ def full_history_details
       sexual_orientation: sexual_orientation || ''
     }
   end
+
+  def to_h
```

Comment:
Why not just put this on the model? Seems hidden here.

Reply:
I put it here because the other "data manipulation before sending to frontend" methods are here but I'll move it.